File: swappy/src/swap/MQExample/java/Send.java
import com.rabbitmq.client.ConnectionFactory;
import com.rabbitmq.client.Connection;
import com.rabbitmq.client.Channel;

public class Send {
	private final static String QUEUE_NAME = "hello";

	public static void main(String[] argv) throws java.io.IOException {

		ConnectionFactory factory = new ConnectionFactory();
	    factory.setHost("157.253.220.158");
	    //factory.setHost("localhost");
	    factory.setPort(5672);
	    factory.setUsername("guest");
	    factory.setPassword("admin1234");
	    Connection connection = factory.newConnection();
	    Channel channel = connection.createChannel();
	    channel.queueDeclare(QUEUE_NAME, false, false, false, null);
	    String message = "Hello World!";
	    channel.basicPublish("", QUEUE_NAME, null, message.getBytes());
	    System.out.println(" [x] Sent '" + message + "'");
	    channel.close();
    	connection.close();
    	
  	}
}

File: swappy/src/swap/MQExample/python/send.py
#!/usr/bin/env python
import pika

connection = pika.BlockingConnection(pika.ConnectionParameters(
               'localhost'))
channel = connection.channel()
channel.queue_declare(queue='hello')
while True:
	body = raw_input("Message: ")
	channel.basic_publish(exchange='',
	                      routing_key='hello',
	                      body=body)
	print " [x] Sent ", body
connection.close()
